gcpvault.go

```diff
@@ -60,8 +61,40 @@ type Config struct {
 	// HTTP requests made by this library. If not set, an http.Client with a 1s
 	// IdleConnTimeout will be used.
 	HTTPClient *http.Client
+
+	TokenCache TokenCache
+	// How long before the token expiration should be it be regenerated (in minutes).
+	// Default is 5 min.
+	CachedTokenRefreshThreshold int `envconfig:"TOKEN_CACHE_REFRESH_THRESHOLD"`
```

Comment:
should these match?  i.e. `TokenCacheRefreshThreshold` instead?

Reply:
agree